### ISSUE TITLE
Any progress on the fans?

### ISSUE THREAD

**Issue by wagenadl** [pelrun/hp-omen-linux-module#24]
Module mostly works on OMEN by HP Laptop 17-ck0xxx:

* Very happy to set the colors to something less garish. 
* The Omen key is available to xkb

What doesn't work:

* The P1 to P6 keys are not available to xkb.

Any progress on the fan control? That would be my major request: my laptop keeps its fan on all the time at a low speed and I'd much prefer silence...

Thanks!

**Comment by Illuminat10n** [comment]
For me nbfc works for fan control.

**Comment by alou-S** [comment]
@I-dont-need-name Omen 15? I have a Omen 16 and nbfc doesn't seem to work

**Comment by Illuminat10n** [comment]
@alou-S yes. I used this config file: https://github.com/hirschmann/nbfc/issues/1227 (Slightly adjusted the values to make it more cool)

**Comment by alou-S** [comment]
For those who come here later I made this Python Script that uses EC to control fans : https://github.com/alou-S/omen-fan
